server/handler/blockex.js

```diff
@@ -148,7 +148,7 @@ const getCoinsWeek = () => {
  */
 const getMasternodes = async (req, res) => {
   try {
-    const limit = req.query.limit ? parseInt(req.query.limit, 10) : 10;
+    const limit = req.query.limit ? parseInt(req.query.limit, 1000) : 1000;
```

Comment:
This changes the Radix of the `parseInt` to 1000, which would return NaN.

Reply:
Merged.